### PR TITLE
i18n: Auth

### LINKED ISSUE #73: i18n: Auth

### PR DESCRIPTION
**Pull request by Stefanacef**
Demo: https://wedance-stefanacef.netlify.app/nopassword/

Closes #73

### CODE REVIEW

**Comment by razbakov** [comment]
@Stefanacef i forced update your auth-translation branch, you will need to delete that branch locally and check it out again.
I had to reset unrelevant changes to avoid conflicts on merge.

File: layouts/default.vue [hunk 1]
```diff
@@ -4,12 +4,14 @@
...
     <TPopup
       v-if="showAuthPopup"
-      title="Members only"
+      :title="$t('popup.title')"
       @close="showAuthPopup = false"
     >
       <div class="my-4 w-64 flex flex-col justify-center text-center">
-        <div class="p-4">Sign in to {{ showAuthPopup }}</div>
-        <TButton class="mt-2" type="primary" to="/signin">Sign in</TButton>
+        <div class="p-4">{{ $t('popup.request') }} {{ showAuthPopup }}</div>
```
[the remark this answers is not in the document]
Comment (best-lucky1030) [reply layouts/default.vue:11]: Yes, that's correct. it comes from the `title` property.  👍

File: layouts/default.vue [hunk 2]
```diff
@@ -4,12 +4,14 @@
   >
     <TPopup
       v-if="showAuthPopup"
-      title="Members only"
+      :title="$t('popup.title')"
```
[the remark this answers is not in the document]
Comment (razbakov) [reply layouts/default.vue:7]: That wasn't updated

**Comment by razbakov** [comment]
@Stefanacef this pull request doesn't contain your changes.

**Comment by Stefanacef** [comment]
yes, I'm working on fixing the situation now :)

**Review by best-lucky1030 — COMMENTED** [review]
I think there will be conflicts when this PR is merged into `main` branch.

**Comment by best-lucky1030** [comment]
resolved the conflit on `en.yml file`

File: locales/en.yml [hunk 3]
```diff
@@ -435,3 +438,10 @@ TInputProfile:
   addInstagram: (add from Instagram)
   addFacebook: (add from Facebook)
   placeholder: Search on WeDance, Instagram or Facebook
+popup:
+  auth:
+    title: Members only
+    body: Sign in to {message}
```
Comment (razbakov) [inline locales/en.yml:444]:
```suggestion
    body: Sign in to {action}
```

File: locales/en.yml [hunk 4]
```diff
@@ -435,3 +438,10 @@ TInputProfile:
...
+    signin: Sign in
+TButton:
+  login: Login to {title}
```
Comment (razbakov) [inline locales/en.yml:447]:
```suggestion
  login: Login to {action}
```

**Review by razbakov — CHANGES_REQUESTED** [review]
Very good! 

One small polishing for translators to better understand the sentence they going to translate.

And one more thing: try to signout and click on different buttons on the Feed:
- star
- watch
- context menu (three dots near post)

They all have different auth message, cuz they have a special `title` attribute on the button.
Find all usage of the `<TButton` in the code and update `title`

**Review by razbakov — APPROVED** [review]

**Review by amanuela97 — APPROVED** [review]

**Pull request merged**